gui/default/index.html

```diff
@@ -329,9 +332,39 @@ <h3 class="panel-title">
         </div>
       </div>
 
-      <!-- First regular row -->
+      <!-- Login form -->
+      <div ng-if="!authenticated" class="center-block">
```

Comment:
Couldn't this be nicely extracted into its own component (or whatever AngularJS calls it) so the form lives in its own HTML document, while being included conditionally?

Reply:
Obviously perfectly fine (probably even nice) to do that, but I don't think it should be a requirement for this PR. As you said, current practice is already very monolithic and imo it's fine to keep going with that.